client/influxdb_test.go

```diff
@@ -741,3 +765,28 @@ war3JNM1mGB3o2iAtuOJlFIKLpI1x+1e8pI=
 		}
 	}
 }
+
+func TestChunkedResponse(t *testing.T) {
+	s := `{"results":[{},{}]}{"results":[{}]}`
+	r := client.NewChunkedResponse(strings.NewReader(s))
+	resp, err := r.NextResponse()
+	if err != nil {
+		t.Fatalf("unexpected error.  expected %v, actual %v", nil, err)
```

Comment:
Same response I always give :) the rest of the file is one way and I'm going to follow whatever the rest of the file does. I think the double space is weird, but keeping an individual file consistent is more important than my OCD.